doc/api/fs.md

```diff
@@ -329,14 +329,17 @@ Stats {
   birthtime: Mon, 10 Oct 2011 23:24:11 GMT }
 ```
 
-Please note that `atime`, `mtime`, `birthtime`, and `ctime` are
+*Note*:
+* `atime`, `mtime`, `birthtime`, and `ctime` are
 instances of [`Date`][MDN-Date] object and appropriate methods should be used
 to compare the values of these objects. For most general uses
 [`getTime()`][MDN-Date-getTime] will return the number of milliseconds elapsed
 since _1 January 1970 00:00:00 UTC_ and this integer should be sufficient for
 any comparison, however there are additional methods which can be used for
 displaying fuzzy information. More details can be found in the
 [MDN JavaScript Reference][MDN-Date] page.
+* For backward compatibile copying, the `toJSON` method can be used:  
+  `const c = s.toJSON ? s.toJSON() : Object.assign({}, s);`
```

Comment:
Use better variable names here to make things clearer, such as `stats` instead of `s` and `copy` instead of `c`.